activerecord/test/cases/tasks/database_tasks_test.rb

```diff
@@ -339,15 +339,14 @@ def teardown
     end
 
     def test_migrate_receives_correct_env_vars
-      verbose, version = ENV["VERBOSE"], ENV["VERSION"]
+      version = ENV["VERSION"]
 
-      ENV["VERBOSE"] = "false"
```

Comment:
I see. In this case I think it would be more useful if we change the test to assert VERBOSE too. WDYT?